cfgov/jinja2/v1/ask-cfpb/landing-page-spanish.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'spanish-ask-base.html' %}
+
+{% block babel_content %}
+<section class="content full-bleed">
+    <div class="primary">                
+        <div class="ac-catnav-row">              
+        {% for cat in categories %}
+            <article class="ac-catnav ac-cat-{{ cat.slug_es }}">
+                <h2 class="ac-catnav-heading"><a href="">
+    			<img src="{{static('nemo/_/img/icon-' + cat.slug + '-retina.png') }}" alt="" class="icon-40">{{ cat.name_es }}</a></h2>
```

Comment:
Minor: Extra indentation, missing space between `{{` and `static`. Please break the `</a></h2>` onto a new line so it matches the opening tags and is easy to identify.